stable/chat/v1/messages.proto

```diff
@@ -111,21 +82,24 @@ message Action {
     string label = 1;
     // Whether this text input should be a multiline one or not.
     bool multiline = 2;
-    // Contextual data allowing the bot to discern what the user input is for.
-    bytes data = 3;
+    // The default text that should be pre-filled in the text field.
+    optional string default = 3;
   }
 
   // Type of the action.
   Type action_type = 1;
+  // Contextual info allowing the bot to discern what the action is for.
+  // This MUST be unique per-message (but can be re-used in different messages).
```

Comment:
"be unique per message" should be expanded to "the info of an action cannot be the same with the info of another action in the same message".